### ISSUE TITLE
Error in DupFinder "Project 'nuspec' root directory doesn't exist"

### ISSUE THREAD

**Issue by pascalberger** [cake-contrib/Cake.Recipe#119]
There still seems to be an issue with latest Cake.Recipe regarding DupFinder.

```
========================================
DupFinder
========================================
Executing task: DupFinder
Deleting file C:/projects/cake-prca/1c2b676f-9eea-44bd-844f-b28f04aeba6f.cake
JetBrains Duplicates Finder 2017.1.2
Running in 64-bit mode, .NET runtime 4.0.30319.42000 under Microsoft Windows NT 6.2.9200.0
Collecting files to analyze:
  from project 'C:\projects\cake-prca\src\Cake.Prca.sln' excluding: C:/projects/cake-prca/src/Cake.Prca.Tests/*.cs.
Cake.exe : Project 'nuspec' root directory doesn't exist. Checked path C:\projects\cake-prca\src\Build
At line:1 char:1
+ & "C:\projects\cake-prca\tools\Cake\Cake.exe" "setup.cake" -target="A ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Project 'nuspec...-prca\src\Build:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
34 files found to analyze.
```

Can be reproduced [here](https://ci.appveyor.com/project/cakecontrib/cake-prca/build/0.5.0-beta.1+1.build.137#L570)

**Comment by pascalberger** [comment]
I assume this is related to the update of the JetBrains tool in a recent version of Cake.Recipe and propably should affect other addins from the cake-contrib organization as well. Since Cake.Recipe script is cached and also the build doesn't break it might not have been noticed yet.

**Comment by gep13** [comment]
I remember seeing this issue at the time, but haven't seen it again recently, and JetBrains tools have undergone major releases since this issue was opened.  Closing for now, feel free to re-open.

**Comment by pascalberger** [comment]
I can confirm that this was fixed with JetBrain tools 2018.3 (see [RSRP-471361](https://youtrack.jetbrains.com/issue/RSRP-471361)), which was updated with #340
